zmap/zgrab2
# Update MSSQL GoDocs

* re-export MSSQL primitives, remove stuttering `mssql.mssqlXYZ`  names
 * re-run golint / gofmt
 * add package godocs
 * add detailed scan docs
 * bugfixes:
  * populate InstanceName in result
  * fix return value in case of failed / incomplete detection
  * add default EncryptMode

## How to Test

`TEST_MODULES=mssql make integration-test`

 * Confirm that InstanceName is set
 * Point to a non-mssql service, like HTTP, and confirm that it returns a nil result
